tests/test_indexer.py

```diff
@@ -63,59 +64,51 @@
 #   5. Verify the structure and content of the index document
 #
 
+class HTTPInfo:
+    address = "127.0.0.1"
+    port = None
+    server = None
+    thread = None
+
+class ESInfo:
+    server = None
+
+def setUpModule():
+    HTTPInfo.port = findOpenPort()
+    HTTPInfo.server = HTTPServer((HTTPInfo.address, HTTPInfo.port), PostTestHandler)
+    HTTPInfo.thread = threading.Thread(target=HTTPInfo.server.serve_forever)
+    HTTPInfo.thread.start()
+
+    ESInfo.server = ElasticsearchServer()
+    os.environ['DSS_ES_PORT'] = str(ESInfo.server.port)
+
+def tearDownModule():
+    ESInfo.server.shutdown()
+    HTTPInfo.server.shutdown()
 
-class TestIndexer(unittest.TestCase, DSSAsserts, StorageTestSupport, DSSUploadMixin):
 
-    http_server_address = "127.0.0.1"
-    http_server_port = 8729
+class TestIndexerBase(DSSAsserts, StorageTestSupport, DSSUploadMixin):
```

Comment:
so why does this not extend `unittest.TestCase`?

Reply:
pyunit will think this contains valid test cases and run it. In that case they would all fail because     `def create_bundle_created_event(self, bundle_key, bucket_name):
def process_new_indexable_object(self, event, logger):`
are not implemented.